compiler/next/lib/parsing/chpl.ypp

```diff
@@ -2412,49 +2414,18 @@ var_decl_type:
 ;
 
 var_decl_stmt:
-  TCONFIG var_decl_type var_decl_stmt_inner_ls TSEMI
-    {
-      $$ = TODOSTMT(@$);
-      context->resetDeclState();
-    }
+  TCONFIG
+  {
+    // Use a mid-rule action to thread along 'isVarDeclConfig'.
+    context->noteIsVarDeclConfig(true);
```

Comment:
Mostly out of curiosity: do we have different `context`s per token? Reading this made me wonder whether we should set this back to false after parsing this rule is complete.

Reply:
Thanks, that's probably a bug if it doesn't do that!

Reply:
Ah, `resetVarDeclState()` in the builder takes care of turning it off.